docs/examples/DataSet/Deeper dive on qcodes doNd functionalities with a focus on arrayed sweeping benefits.ipynb

```diff
@@ -0,0 +1,455 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Deeper dive on qcodes doNd functionalities with a focus on arrayed sweeping benefits"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example notebook, we want to show how the qcodes measurement can be simplified by using qcodes doNd functions. Also, we test 2-dimentional sweeping with a naive point by point sweeping over x and y axis and measure z axis values compared to sweeping over x axis and considering y axis as an array of numbers to measure z axis values. The array method can be useful in buffered data acquisitions from instruments that have buffer to write data in it (this notebook is not aiming to show how to setup buffer). Therefore, the measurement time can be enhanced significantly."
+   ]
```

Comment:
i do suggest to explain/describe what a buffered data acquistion is meant here broader. "instruments that have buffer to write data in it" reads more as instrument's implementation detail rather than approach where you tell the instrument what to sweep and it returns you arrays as opposed to setting/getting parmaeters value by value.